charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
I think there is an issue in the srcipt, it pushed the jupyter web app related templates in kubeflow-pipelines in here

Reply:
it was a mis-conduct when I was pasting the path, removed the jupyter web app content from pipeline chart. Actually I need to regenerate the chart for pipeline s3 only

Reply:
done now @surajkota

Reply:
@jsitu777 what about rds-s3
https://github.com/awslabs/kubeflow-manifests/blob/main/charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/workflow-controller-configmap-kubeflow-ConfigMap.yaml#L11

Reply:
@jsitu777 you need to regenerate chart for all kubeflow pipeline flavour to get in sync with
https://github.com/awslabs/kubeflow-manifests/pull/339/files#diff-402eb2df5dee333f2dc515c95eac04f19594757c7353a3feb15c8bf71aa89ff3R8-R10